backend/payment.php

```diff
@@ -15,11 +16,12 @@
             'amount' => $amount,
             'currency' => 'usd',
             'card' => $token,
-            'description' => 'elementary OS Freya 0.3.2',
+            'description' => $description,
             'receipt_email' => $email,
         ));
         // Set an insecure, HTTP only cookie for 10 years in the future.
-        setcookie('has_paid_freya', $amount, time() + 315360000, '/', '.elementary.io', 0, 1);
+        $encoded = urlencode(str_replace(' ', '', $description));
```

Comment:
It gets the description name from the payment script, which should be the release variable above ('Freya 0.3.2'). How about just adding 'has_paid_' in front of description?